sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -96,7 +96,7 @@ If your application is hosted in Azure, you can make use of [Managed Identity](h
 
 | Credential with example                                                     | Usage                                                                                                                                                                                                                                                                                                                                                                        |
 | --------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| [ManagedIdentityCredential](#authenticating-in-azure-with-managed-identity) | Authenticate in a virtual machine, app service, function app, cloud shell, or AKS environment on Azure, with system assigned, or user assigned managed identity enabled.                                                                                                                                                                                                     |
+| [ManagedIdentityCredential](#authenticating-in-azure-with-managed-identity) | Authenticate in a virtual machine, app service, function app, cloud shell, or AKS environment on Azure, with system assigned managed identity, user assigned managed identity, or app registration. .                                                                                                                                                                                                     |
```

Comment:
App registrations aren't universally supported. Only AKS pod identity supports using an arbitrary service principal in addition to a user assigned identity. Should we call out this detail in these docs?

Reply:
@schaabs I’ve added more information regarding AKS. Please review again!